tests/doc_scan_service/session/retrieve/third.party.identity.fraud.1.check.response.spec.js

```diff
@@ -0,0 +1,88 @@
+const ThirdPartyIdentityFraud1CheckResponse = require('../../../../src/doc_scan_service/session/retrieve/third.party.identity.fraud.1.check.response');
+const GeneratedProfileResponse = require('../../../../src/doc_scan_service/session/retrieve/generated.profile.response');
+const GeneratedMedia = require('../../../../src/doc_scan_service/session/retrieve/generated.media');
+const ReportResponse = require('../../../../src/doc_scan_service/session/retrieve/report.response');
+
+describe('ThirdPartyIdentityFraud1CheckResponse', () => {
```

Comment:
Following my comment about this `ThirdPartyIdentityFraud1CheckResponse` class could extend `ProfileCheckResponse`, and given that the `ProfileCheckResponse` test are already doing what is done within this test file, I think we could simply assert the here that the inheritance is in place.

Reply:
I've removed all of the tests related to the `GeneratedProfileResponse` and also the ones inherited from `CheckResponse`. There is a single test that is asserting that the `.getGeneratedProfile` method returns an instance of `GeneratedProfileResponse`. Not sure if the intention is to keep this?